examples/ibm-is-vpc/variables.tf

```diff
@@ -18,3 +18,5 @@ variable "profile" {
   default = "bc1-2x8"
 }
 
+variable "lb_name" {
```

Comment:
Don't take name of the load balancer from user..Use the load balancer created in the example it self